src/utils/extractTimeFromDate.ts

```diff
@@ -0,0 +1,12 @@
+export function extractTimeFromDate(dateString: Date): string {
```

Comment:
please comments to all your utils funtions, examle of input and output params, its will be easier for other team members reuse it in case it will be needed. something similar to this one https://www.google.com/search?sca_esv=588105669&sxsrf=AM9HkKmttWTagYTObzQ55lUQ9adRT2Ts5A:1701802729456&q=how+comment+code+right+for+function&tbm=isch&source=lnms&sa=X&ved=2ahUKEwjSz6qZ_fiCAxWPIRAIHYKNC9cQ0pQJegQICxAB&biw=1440&bih=728&dpr=2#imgrc=3hdkHotkPJgM9M